action/README.md

```diff
@@ -0,0 +1,66 @@
+# statick-action
+
+![Statick](https://github.com/triboelectric/statick-action/workflows/Statick/badge.svg)
```

Comment:
We'll need to update this URL.

Reply:
Thanks for catching this.  I fixed the PR branch version, [viewable here](https://github.com/kogut/statick/blob/525-move-statick-action-into-this-main-repo/action/README.md). 

- [ ] Change to expected `main` version just before merge.